.github/workflows/build-python.yml

```diff
@@ -5,7 +5,7 @@ on:
   push: # Apply to all branches
 
 env:
-  PYTHON_VERSION: 3.8.10
```

Comment:
I would expect the python version to be 3.8.10. Why did you change this? It was not requested as I know.

Reply:
If I understand it correctly there was a migration to the latest ubuntu 22.04 and the "build-python" step failed with:

The version '3.8.10' with architecture 'x64' was not found for Ubuntu 22.04.
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

Reply:
as discussed, setting the python version to 3.8 is not critical and has no direct effect on our RBT release